renovate.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:base"],
+  "includePaths": ["packages/create-plugin/templates/**"],
+  "separateMajorMinor": false,
+  "enabledManagers": ["regex"],
+  "customManagers": [
+    {
+      "customType": "regex",
+      "fileMatch": ["_package.json"],
+      "matchStrings": ["\"(?<depName>@grafana/.*)\"[^\"]*:\\s*\"\\^(?<currentValue>.*)\""],
+      "datasourceTemplate": "npm"
+    },
+    {
+      "customType": "regex",
+      "fileMatch": ["plugin.json"],
+      "matchStrings": ["\"grafanaDependency\":\\s*\"\\>=(?<currentValue>.*)\""],
+      "depNameTemplate": "grafana/grafana-enterprise",
+      "datasourceTemplate": "docker"
+    },
+    {
+      "customType": "regex",
+      "fileMatch": ["(^|/)(?:docker-)?compose[^/]*\\.ya?ml$"],
+      "matchStrings": ["grafana_version:\\s\\$\\{GRAFANA_VERSION:-(?<currentValue>.*)\\}"],
+      "depNameTemplate": "grafana/grafana-enterprise",
+      "datasourceTemplate": "docker"
+    }
+  ],
+  "packageRules": [
+    {
+      "matchPackagePatterns": ["*"],
```

Comment:
Maybe I'm reading this incorrectly but how does this match to all grafana dependencies? Because it's only gonna run against the above customManagers?

Reply:
Yes I think these rules only applies to the custom managers above, but let's change this to `@grafana/*` just to be sure.

Reply:
Follow up: needed to specify `grafana/grafana-enterprise` package here too. If not, Renovate would have created two separate PRs.